### ISSUE TITLE
Remove URL parameter info in sidebar and move it to the wiki

### ISSUE THREAD

**Issue by alschmut** [MaibornWolff/codecharta#589]
# Feature request

## Description

- Remove the *URL Parameter* button from the sidebar. 
- Move the file parameter information from our [visualization README](https://github.com/MaibornWolff/codecharta/tree/master/visualization#url-parameters-used-by-the-web-application)  to the wiki.
- Add information about the same-origin policy when loading maps by parameter and how to work with it. 

<img width="40%" src="https://user-images.githubusercontent.com/23529226/59933374-42710900-9441-11e9-9139-da2525f9e59d.jpg">

<img src="https://user-images.githubusercontent.com/23529226/59933375-42710900-9441-11e9-954c-89be23350885.jpg">

**Comment by Richargh** [comment]
Let's move it to our wiki. I don't think any users need this information. It's mostly useful for developers that want to load a map into CC. The devs then also need some tipps for how to deal with same-origin policy. I changed this issue to also be a documentation issue.

**Comment by alschmut** [comment]
See new wiki page https://github.com/MaibornWolff/codecharta/wiki/Load-map-using-URL-Parameters.

**Comment by alschmut** [comment]
But I don't know, what you want to be documented with:
> The devs then also need some tipps for how to deal with same-origin policy

**Comment by Richargh** [comment]
> But I don't know, what you want to be documented with:

Maybe @NearW can chime in here. I noticed a lot of confusion in our group, when we tried to get the Jenkins plugin to work. The same-origin policy made loading maps into CC very tricky. 

In browser speak an origin consists of **protocol**://**domain**:**port**/whatever. Everything but whatever needs to match for CC to load a map. Unless the right CORS-headers are set. I'm sorry if this sounded preachy but again it created a lot of confusion for us and I wanted this standard behavior documented just in case. It does not have to be longer than this but it could reduce a lot of confusion for for future users.
